demo/util/util.mjs

```diff
@@ -0,0 +1,14 @@
+/**
+ * Replaces |swapOut| with |swapIn| in the DOM. Reparent all children
+ * and copy all attributes.
+*/
+export function swap(swapOut, swapIn) {
+  while (swapOut.childNodes.length > 0) {
+    swapIn.appendChild(swapOut.firstChild);
+  }
```

Comment:
I think this loop can be replaced with `swapIn.append(...swapOut.childNodes)`.

Reply:
cool.